benefits/core/templates/core/includes/form.html

```diff
@@ -3,41 +3,39 @@
 
   {% url form.action_url as form_action %}
 
-  <form id="{{ form.id }}" action="{{ form_action }}" method="{{ form.method | default:"post" | upper }}" role="form">
+  <form id="{{ form.id }}"
+        class="{{ form.classes }}"
+        action="{{ form_action }}"
+        method="{{ form.method | default:"post" | upper }}"
+        role="form">
     {% csrf_token %}
 
-    <div class="row form-container justify-content-center">
-      <div class="{{ form.classes }}">
```

Comment:
I think there's a regression in Benefits Admin coming from this change.

We have some styles in `admin/styles.css` so that descendant checkboxes will be on the left side: https://github.com/cal-itp/benefits/blob/dce769c917469c7e9f1fb4f781e339005437e772/benefits/static/css/admin/styles.css#L88-L103

`InPersonEligibilityForm` has `checkbox-parent` in its `classes`, which would end up on this line, but now it's on the `<form>` element. 

| main | this branch (with #2555 merged in locally... *)|
| --- | --- |
| ![image](https://github.com/user-attachments/assets/908d41d2-8503-4fd4-aec6-6b8701cc161e) | ![image](https://github.com/user-attachments/assets/08aa4cf7-e77c-42d4-8d79-9ad313c69257)  |

Not sure if we should put this `<div>` back or try to tweak the CSS?

(*When we're back in the new year, it'd be good to get all this reCAPTCHA work caught up with the fixes that are in main.)